ddtrace/contrib/botocore/utils.py

```diff
@@ -193,23 +193,11 @@ def set_response_metadata_tags(span, result):
 
 def extract_DD_context(messages):
     ctx = None
-    if len(messages) == 1:
+    if len(messages) >= 1:
         message = messages[0]
         context_json = extract_trace_context_json(message)
```

Comment:
Could a payload messages with different trace contexts? If this is a valid scenario should we try to reconcile differences (maybe through span links)?

Is there a valid scenario where the trace context is only sent in the 2nd, 3rd, 4th, .... message?

Reply:
This is a valid scenario, but not one that we're targeting due to the limitations of current trace context propagation. For now, we want to be consistent with other languages, which will only use the first message's context. We will address these problems once we get to the implementation of span links